fspiop-api/documents/API Definition v2.0-alpha.md

```diff
@@ -3075,11 +3070,11 @@ If an FSP issues an instruction to execute a conversion using a conversion quota
 
 ##### 6.13.2.3 Timeout and Expiry
 
-The requesting FSP must always set a transfer expiry time to allow for use cases in which a swift completion or failure is needed. If the use case does not require a swift completion, a longer expiry time can be set.
+The FSP requesting the conversion must always set a transfer expiry time to allow for use cases in which a swift completion or failure is needed. If the use case does not require a swift completion, a longer expiry time can be set.
 
-The arbiter of the expiry of a conversion request is the switch; or, in the case where a switch is not present, the FXP. A single point of evaluation for timeout times is required because participants in a Mojaloop system may be relatively small and informal institutions and the implementation of a system to prevent clock drift over a whole Mojaloop scheme may be challenging.
+The arbiter of the expiry of a conversion request is the switch; or, in the case where a switch is not present, the FXP. A single point of evaluation for timeout times is required because the communicating participants may be relatively small and informal institutions and the implementation of a system to prevent clock drift among all participants may be challenging.
```

Comment:
Wouldn't it be easier to understand if "FSPs" is used instead of "communicating participants"?